clang/include/clang/Basic/Attr.td

```diff
@@ -2256,12 +2256,12 @@ def SYCLIntelPipeIO : InheritableAttr {
 }
 
 // Variadic integral arguments.
-def IntelFPGABankBits : Attr {
+def IntelFPGABankBits : InheritableAttr {
   let Spellings = [CXX11<"intel", "bank_bits">];
   let Args = [VariadicExprArgument<"Args">];
   let Subjects = SubjectList<[IntelFPGAConstVar, IntelFPGALocalStaticAgentMemVar,
                               Field], ErrorDiag>;
-  let LangOpts = [SYCLIsDevice, SYCLIsHost];
+  let LangOpts = [SYCLIsDevice, SilentlyIgnoreSYCLIsHost];
```

Comment:
This changes behavior right? Was attribute handled on host before? Prior to this change, it looks like it was accepted. I'm not sure if it was handled.

Reply:
I do not think it changes any behavior. None of the memory attributes currently support on Host.  Yes, it was accepted before but it did not have any affect on Host.

Reply:
Thanks for clarifying!